.github/workflows/ai-generated-release-notes.yml

```
@@ -0,0 +1,388 @@
+      - name: Check if this is CodeRabbit's first comment
+        id: check-first-comment
+        uses: actions/github-script@v7
+        with:
+          script: |
+            try {
+              const { data: comments } = await github.rest.issues.listComments({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                issue_number: context.issue.number
+              });
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)